praw/models/reddit/subreddit.py

```diff
@@ -882,6 +882,12 @@ def edited(self, only=None, **generator_kwargs):
         Additional keyword arguments are passed in the initialization of
         :class:`.ListingGenerator`.
 
+        To print all items in the edited queue try:
+        .. code:: python
+
+          for item in reddit.subreddit('mod').mod.edited(limit=None):
```

Comment:
From [here](http://praw.readthedocs.io/en/latest/code_overview/other/subredditmoderation.html#praw.models.reddit.subreddit.SubredditModeration.inbox) I did 2 spaces instead of 3 and it worked. But I did set all the other remaining ones to 3 spaces and going to update that one linked to 3 as well for consistency.